src/ui/features/onboarding/Success/styles.module.css

```diff
@@ -10,7 +10,7 @@
 }
 
 .container {
-  background-image: url('../assets/success_background.png');
+  background-image: url('~src/ui/assets/success_background.png');
```

Comment:
Now it is shared (since we have 2 `Success` components that use this image), so I moved it to `src/ui/assets`

Reply:
We will need to check this in production build

Reply:
Thanks, indeed! Added to the test plan in the PR description